services/dynamic-sidecar/docker/entrypoint.sh

```diff
@@ -110,4 +110,11 @@ echo "  $SC_USER_NAME rights    : $(id "$SC_USER_NAME")"
 echo "  local dir : $(ls -al)"
 echo "  volumes dir : $(ls -al "${DYNAMIC_SIDECAR_DY_VOLUMES_MOUNT_DIR}")"
 
+echo "$INFO" "Available permissions at etntrypoint finish"
+capsh --print
+
+echo "$INFO" "Granting /usr/local/bin/python3.10 CAP_LINUX_IMMUTABLE"
```

Comment:
is there no way of getting the current python instead of hard-coding 3.10?
there is at least VIRTUAL_ENV.. maybe check for the PYTHON one...